src/components/typography/typography.stories.mdx

```diff
@@ -134,14 +135,14 @@ It's also possible to create ordered and unordered lists using the `List`, `List
 | errorRed              | ✅               | ✅             | Negative text      |
 
 ## Props
-
-<ArgsTable
-  rows={[
-    {
-      name: "variant",
-      type: {
-        summary:
-          "'h1-large' | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'segment-header' | 'segment-header-small' | 'segment-subheader' | 'segment-subheader-alt' | 'p' | 'small' | 'big' | 'sup' | 'sub' | 'strong' | 'b' | 'em'",
+### Typography
```

Comment:
This component is no longer appearing in the Design System components list in the Storybook left hand nav